ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -2174,7 +2203,31 @@
                 "\n                                                 AmountType = sc.Identity.AmountType,",
                 "\n                                                 Value = sc.Value,",
                 "\n                                                 Partition = GetStorage().TargetPartition });",
-                "\n                            ",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public interface TmToIfrsVariable: IScope<ImportIdentity, ImportStorage>",
+                "\n{",
+                "\n    private string EconomicBasis => Identity.ValuationApproach == ValuationApproaches.VFA ? EconomicBases.C : EconomicBases.L;",
+                "\n    protected IEnumerable<IfrsVariable> amortizationFactor => Identity.AocType == AocTypes.AM",
```

Comment:
A point worth noticing is that I changed the access modifier here from **private** to **protected**

Reply:
This is fine, otherwise you cannot access the amortizationFactor in the scopes below.